pom.xml

```diff
@@ -52,9 +52,8 @@
 
     <nukleus.plugin.version>0.10</nukleus.plugin.version>
     <nukleus.version>0.6.2</nukleus.version>
-
-    <nukleus.http.spec.version>0.30</nukleus.http.spec.version>
     <reaktor.test.version>0.10</reaktor.test.version>
+    <nukleus.http.spec.version>develop-SNAPSHOT</nukleus.http.spec.version>
```

Comment:
Please don't change the property order - I'm trying to layout the POMs consistently across projects.